plugins/lang/mapchooser.txt

```diff
@@ -9,11 +9,11 @@ TIME_CHOOSE = It's time to choose the nextmap...
 
 [de]
 CHO_FIN_EXT = Auswahl beendet. Laufende Map wird um %.0f Minuten verlaengert.
-CHO_FIN_NEXT = Auswahl beendet. Naechste Map ist %s
-CHOSE_EXT = %s waehlten Map-Verlaengerung
-X_CHOSE_X = %s waehlten %s
-CHOOSE_NEXTM = AMXX waehlt naechste Map
-EXTED_MAP = Verlangere Map %s
+CHO_FIN_NEXT = Auswahl beendet. Naechste Map ist: %s
+CHOSE_EXT = %s waehlte: Map-Verlaengerung
+X_CHOSE_X = %s waehlte: %s
```

Comment:
I don't think any of these need the colon. The move to singular is enough.

Reply:
Okay, I agree, will be changed